src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator.UnitTests/Verifiers/CSharpSourceGeneratorVerifier`1+Test.cs

```diff
@@ -31,28 +31,17 @@ namespace Microsoft.CodeAnalysis.ResxSourceGenerator.Test
     public static partial class CSharpSourceGeneratorVerifier<TSourceGenerator>
         where TSourceGenerator : IIncrementalGenerator, new()
     {
-        public class Test : CSharpSourceGeneratorTest<TSourceGenerator, XUnitVerifier>
+        public class Test(string identifier, [CallerFilePath] string? testFile = null, [CallerMemberName] string? testMethod = null) : CSharpSourceGeneratorTest<TSourceGenerator, XUnitVerifier>
         {
-            private readonly string _identifier;
-            private readonly string? _testFile;
-            private readonly string? _testMethod;
+            private readonly string _identifier = identifier;
+            private readonly string? _testFile = testFile;
+            private readonly string? _testMethod = testMethod;
 
             public Test([CallerFilePath] string? testFile = null, [CallerMemberName] string? testMethod = null)
                 : this(string.Empty, testFile, testMethod)
             {
             }
 
-            public Test(string identifier, [CallerFilePath] string? testFile = null, [CallerMemberName] string? testMethod = null)
-            {
-                _identifier = identifier;
-                _testFile = testFile;
-                _testMethod = testMethod;
-
-#if WRITE_EXPECTED
-                TestBehaviors |= TestBehaviors.SkipGeneratedSourcesCheck;
-#endif
```

Comment:
Looks like the conversion to primary constructors nuked this code.

Reply:
Oh you address this later in fa45598bd6d27daa3a0bd6e98e021714d3751a2e. Is this a bug for Roslyn?

Reply:
Yes, fixed by @CyrusNajmabadi in https://github.com/dotnet/roslyn/pull/68612 yesterday.